src/filebrowser/FilePicker.vue

```diff
@@ -163,6 +163,11 @@ export default {
 			}
 		} catch(e) {
 			console.log('error retriving project', e)
+			if (e.response.status == 401) {
+				// there was a permission error
+				// we should redirect the user to login
+				this.$router.push('home')
```

Comment:
This redirects to /home, which does not exist, and then gets the default redirect to /.
To redirect to named route the parameter needs to be `{name: "home"}`.

We might want to consider using {name: "home", params: {missingToken: true}} so the user gets the red message box "You are currently not logged in. Please login to access Qvain.". That way it might feel less weird for the user that they are suddenly on the welcome page. The missingToken parameter name could be better, maybe the parameter could be renamed to missingLogin or loginNeeded or something.

Reply:
ack

Reply:
fixed